### DISCUSSION TITLE
Admin warning notification toggle

### DISCUSSION THREAD

**Discussion by BlafKing** [ppy/osu#23986] [Ideas] [answered]
Hi all,

I was thinking it would be a nice addition to have a toggle in the settings for the startup warning message that shows if osu is ran as adminstrator.

I personally use AtlasOS on an old PC to play osu!lazer which results in the message always popping up at start up.
It's not a huge deal, but would be a nice QOL feature if the warning could be toggled on/off in the settings! :)

**Comment by soopax** [comment, marked as answer]
Hello, there is a good reason for the warning. You should not need admin/elevated privileges to run regular programs. Instead, **please create and use a normal account** as it's more secure.

For more information, please read https://en.wikipedia.org/wiki/Principle_of_least_privilege

Thank you.

**Comment by BlafKing** [comment]
Hi,
I understand the reasoning behind the warning, I was just hoping that the option to disable the warning could be added for users that have to run osu!lazer as admin
(when using AtlasOS for example, you cannot disable admin rights, there is a single level of privilege on the OS which is admin.)

I understand it seems like a bit of a silly request, since osu doesn't require admin privileges, but thanks either way for the response! :)

**Reply by Rekunan** [reply]
That sounds like AtlasOS is a massive security risk, and from what I've seen and heard in my quick research, I'd advise against using it.

**Reply by BlafKing** [reply]
Completely agree, If you care a lot about security AtlasOS is definitely not the way to go.
But it helps a great deal when it comes to OS performance and for me personally it's completely worth it :)

**Reply by Rekunan** [reply]
As long as you fully understand all the risks, which it seems like you do, then yeah I guess it works fine. Personally, though, I'd just go for a Linux distro for both security and performance.
